doc(release): prepare release notes for modules 23.04 (#2477)
centreon/centreon-documentation: versioned_docs/version-23.04/releases/centreon-commercial-extensions.md

## Centreon MAP

### 23.04.1

Release date: `July 20, 2023`

#### Enhancements

- [Licensing] Improved automatic license renewal.

> - [UI] Make metric link modal labels more explicit by replacing "metric 1" and "metric 2" by "metric in" and "metric out".

Reviewer: ```suggestion
- [UI] Made metric link modal labels more explicit by replacing "metric 1" and "metric 2" by "metric in" and "metric out".
```

- [UI] You can now hide the label of a resource.
- [UX] You can now duplicate maps.
- [Viewer] You can now define a label for a URL shape.
- [Widget] Fixed incorrect behavior when saving the MAP widget.

#### Bug fixes

> - [Editor] Fixed a bug where empty metric names were blocking edition.

Reviewer: ```suggestion
- [Editor] Fixed a bug where empty metric min or max prevented to save a map.
```

> - [Editor] Fixed metric links when no metric name is filled in.

Reviewer: ```suggestion
- [Editor] Fixed metric links when no metric name was filled in.
```

> - [Server] Enable to parse metric from Broker NEB packets.

Reviewer: ```suggestion
- [Server] Made it possible to parse metrics from Broker NEB packets.
```

- [Server] Fixed a critical error when editing a Geo view.
- [Server] Fixed an issue causing MAP server to crash at start up due to service group loading.

> - [Server] Fixed an issue preventing to save a map with "not a diagram" error message.

Reviewer: ```suggestion
- [Server] Fixed an issue preventing users from saving a map with a "not a diagram" error message.
```

> - [Server] Fixed an issue which made diagnostic to raise an error if a custom URI is used.

Reviewer: ```suggestion
- [Server] Fixed an issue which made diagnostic.sh raise an error if a custom URI was used.
```

- [UI] Fixed cosmetic issues on breadcrumb chips.
- [UI] Fixed the selection of layers in Geo views.
- [UI] Fixed view freezes beyond 500 elements.

> - [Viewer] Fixed a regression causing output not to be displayed in widget output.

Reviewer: ```suggestion
- [Viewer] Fixed a regression causing output not to be displayed in the output widget.
```

> - [Viewer] Fixed an issue making links not to be reactivated when the associated resource goes back on line.

Reviewer: ```suggestion
- [Viewer] Fixed an issue that caused links not to be reactivated when the associated resource went back online.
```

## Centreon BAM

### 23.04.2

Release date: `July 20, 2023`

#### Enhancements

> - [Rules] Fixed boolean rules when resource contains boolean operator in its name.

Reviewer: ```suggestion
- [Rules] Fixed boolean rules when resource contained a boolean operator in its name.
```

#### Bug fixes

> - [Rules] Fixed boolean rules when resource contains boolean operator in its name.

Reviewer: in enhancements

- [UI] Fixed the column name display in the acknowledgment description pop-up.

> - Fixed retrieving list of BAs for users with ACL.

Reviewer: ```suggestion
- Fixed how lists of BAs were retrieved for users with ACLs.
```

## Centreon MBI

### 23.04.1

Release date: `July 20, 2023`

#### Enhancements

- [Licensing] Improved automatic license renewal.

#### Bug fixes

> - [Install] Added missing Gorgone dependencies required by MBI ETL.

Reviewer: ```suggestion
- [Install] Added missing Gorgone dependencies required by the MBI ETL.
```

> - [Reporting] Fixed the display of ongoing alert duration in Hostgroups-Service-Current-Events report.

Reviewer: ```suggestion
- [Reporting] Fixed the display of the duration of ongoing alerts in the Hostgroups-Service-Current-Events report.
```

- [UI] Added CSS in global style files to fix the MBI widget visibility.

> - [UI] Fixed empty "Current usage" column in mbi-storage-list-near-saturation widget.

Reviewer: ```suggestion
- [UI] Fixed empty "Current usage" column in the mbi-storage-list-near-saturation widget.
```
